### PR TITLE
add bn_dict.txt

### PR DESCRIPTION
**Pull request by taeefnajib**

### CODE REVIEW

**Comment by taeefnajib** [comment]
@GreatV please check the PR

File: ppocr/utils/dict/bn_dict.txt [hunk 1]
```diff
@@ -474,4 +474,4 @@ z
 ৬
 ৭
 ৮
-৯
\ No newline at end of file
+৯
```
Comment (GreatV) [inline ppocr/utils/dict/bn_dict.txt:477]:
```suggestion
৯

```

**Review by GreatV — APPROVED** [review]
LGTM

**Comment by taeefnajib** [comment]
Thank you @luotao1 Much appreciated! 🙏

**Comment by dynamicguy** [comment]
Looking forward to it.

**Pull request merged**